napari/_qt/qt_viewer.py

```diff
@@ -194,9 +197,10 @@ def __init__(self, viewer: Viewer, show_welcome_screen: bool = False):
             'napari:toggle_console_visibility',
             self.viewerButtons.consoleButton,
         )
-        action_manager.bind_shortcut(
-            'napari:toggle_console_visibility', 'Control-Shift-C'
-        )
+
+        for action, shortcuts in SETTINGS.shortcuts.shortcuts.items():
+            for shortcut in shortcuts:
+                action_manager.bind_shortcut(action, shortcut)
```

Comment:
So one question is can/should an action have multiple shortcuts, and when we bind a new shortcut do we want to unbind the default one with `action_manager.unbind_shortcut(action)`.

Reply:
I'm guessing this will also be the right place to load translated shortcuts ?

Reply:
I was wondering about the multiple shortcuts.  It seems that some actions had multiple shortcuts before, so I would assume we'd keep it that way?  I think something will have to change to accommodate the multiple shortcuts per action, right?